Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.h

```diff
@@ -171,6 +171,11 @@ namespace AzToolsFramework
             bool IsCyclicalDependencyFound(
                 InstanceOptionalConstReference instance, const AZStd::unordered_set<AZ::IO::Path>& templateSourcePaths);
 
+            //! Checks if it is an override edit on the focused instance.
+            //! \param The given owning instance to be checked.
+            //! \return True if it is an override editing.
+            bool IsOverrideEditing(const Instance& owningInstance) const;
```

Comment:
IsFocusedInstance might be a better name for this and I think PrefabFocusHandler is a better place for this function.